src/allmydata/web/status.py

```diff
@@ -868,73 +868,113 @@ def render_server_timings(self, ctx, data):
             l[T.li["[%s]: %s" % (server.get_name(), times_s)]]
         return T.li["Per-Server Response Times: ", l]
 
-class MapupdateStatusPage(rend.Page, RateAndTimeMixin):
-    docFactory = getxmlfile("map-update-status.xhtml")
 
-    def __init__(self, data):
-        rend.Page.__init__(self, data)
-        self.update_status = data
+# Renders "/status/mapupdate-%d"
+class MapupdateStatusPage(MultiFormatResource):
 
-    def render_started(self, ctx, data):
-        started_s = render_time(data.get_started())
-        return started_s
+    def __init__(self, update_status):
+        """
+        :update_status servermap.UpdateStatus: server map stats provider.
```

Comment:
should be `:param servermap.UpdateStatus update_status:` I think .. at least it's missing `param`

Reply:
Added `:param` in 8e17d20.